packages/scripts/src/launch-http-server.ts

```diff
@@ -35,7 +35,7 @@ export async function launchHttpServer({
 
     app.use('/favicon.ico', noContentHandler);
 
-    const socketServer = io(httpServer, socketServerOptions);
+    const socketServer = new io.Server(httpServer, { ...socketServerOptions, cors: {} });
```

Comment:
This blocks the user from setting their own cors value, perhaps we should default to {} only of no cors key on socketServerOptions?